README.md

```diff
@@ -176,7 +212,7 @@ Multiscrape also offers a `get_content` and a `scrape` service. `get_content` re
 `scrape` does what it says. It scrapes a website and provides the sensors and attributes.
 
 Both services accept the same configuration as what you would provide in your configuration yaml (what is described above), with a small but important caveat: if the service input contains templates, those are automatically parsed by home assistant when the service is being called. That is fine for templates like `resource` and `select`, but templates that need to be applied on the scraped data itself (like `value_template`), cannot be parsed when the service is called. Therefore you need to slightly alter the syntax and add a `!` in the middle. E.g. `{{` becomes `{!{` and `%}` becomes `%!}`. Multiscrape will then understand that this string needs to handled as a template after the service has been called.\
-*If someone has a better solution, please let me know!*
+_If someone has a better solution, please let me know!_
```

Comment:
This is the same, isn't it? :)

Reply:
Yes, but it was done automatically. Not sure if the new gates you introduced or one of extensions I have installed.